File: geohelp-organizer-app/src/app/api/pull-emergencies-organizer-request.ts
export class PullEmergenciesOrganizerRequest {

  public eventID: number;
  public passphrase: string;
  public emergencyID: number;


  constructor(eventID: number, passphrase: string, emergencyID: number) {
    this.eventID = eventID;
    this.passphrase = passphrase;
    this.emergencyID = emergencyID;
  }
}

File: geohelp-backend/src/main/java/ch/geohelp/geohelpbackend/api/model/PullEmergenciesOrganizerRequest.java
package ch.geohelp.geohelpbackend.api.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter

public class PullEmergenciesOrganizerRequest {
    long eventID;
    String passphrase;
    long emergencyID;
}
